### DISCUSSION TITLE
SINGLE COLUMN left/right CLIP SCROLLING

### DISCUSSION THREAD

**Discussion by worker-gray** [SynthstromAudible/DelugeFirmware#2043] [Ideas]
I often need to put a linear automation in very precise locations that often lie on different pages
For example, the 15th note on page 1 and the 2nd note on page 2 of a clip.
Right now, the only way to accomplish this is to make a 2 point automation on page one and then another 2 point automation on page 2.
OR
to zoom out to make both pages visible, but thus sacrificing the accuracy of the 1-to-1 column visibility.
My suggestion is to make an alternate  left to right (horizontal?) scrolling method where instead of flipping between pages at current zoom level, that it could be scrolled per-column.
It would likely need some kind of modifier or button combination.

**Comment by Dust-r** [comment]
Yes! L-R scrolling generally would be awesome on the Deluge. It's hard to make notes sustain longer than the page I'm on.

**Reply by ok-reza** [reply]
note sustaining across pages can already be done by noting which row the note is sequenced on, moving to the page you want it to sustain to, then pressing and holding `<>` and placing the end note anywhere along that note's row.

in regards to OP, maybe a similar logic can be applied using the same shortcut. or simply allowing you to hold 1 point of automation and enable scrolling to other pages and then making you're second point there. since pad + turn `<>` has no function in automation view currently, I could see that working there.

**Reply by worker-gray** [reply]
the preference would be to have start and endpoint on screen, but I would absolutely do it any way possible. your way might even be better since you could conceivably cross multiple screens without changing the resolution.
